examples/resources/xray_licenses_report/resource.tf

```diff
@@ -0,0 +1,27 @@
+resource "xray_licenses_report" "report" {
+
+  name 							= "test-license-report"
```

Comment:
Let's clean up the HCL (blank lines, '=' alignment, etc.) so that the doc looks good.

Same for the other .tf files.

See [Style Conventions](https://developer.hashicorp.com/terraform/language/syntax/style)

Reply:
Fixed, I've used `terraform fmt` and added empty lines manually, according to the convention.